### PR TITLE
Enchance kanban layout UI 

### LINKED ISSUE #176: Enhance Kanban UI Layout

### PR DESCRIPTION
**Pull request by amitbadala**
Resolve #176

image

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,10 +4,11 @@ import Navbar from './components/Navbar/Navbar';
 import Board from './containers/BoardLayout/BoardLayout';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Footer from './components/Footer/Footer';
-
+import KanbanLayout from './containers/KanbanLayout/KanbanLayout';
 function App() {
   return (
     <>
+      {/* <KanbanLayout boardid={'5cfaf71d26607358e66c1d49'} /> */}
```
Comment (rajat-sr) [inline client/src/App.js]: Delete this

File: client/src/components/NewIssue/NewIssue.js [hunk 2]
```diff
@@ -37,36 +37,53 @@ export class NewIssue extends React.Component {
...
+          icon={'add'}
+          minimal={true}
+          onClick={() => this.handleCreateNewIssue()}
+        />
+      );
+    };
     return (
       <Card className={classes.NewIssue}>
-        <EditableText
+        {/* <EditableText
```
Comment (rajat-sr) [inline client/src/components/NewIssue/NewIssue.js]: Delete this

File: client/src/components/NewIssue/NewIssue.js [hunk 3]
```diff
@@ -37,36 +37,53 @@ export class NewIssue extends React.Component {
...
+        <InputGroup
+          className={classes.addIssueInput}
+          placeholder="New Issue..."
+          fill={true}
+          value={this.state.newIssueText}
+          onChange={event => this.handleTextChange(event)}
+          rightElement={addIssue()}
         />
-        <Button text="Add issue" onClick={() => this.handleCreateNewIssue()} />
+        {/* <Button text="Add issue" onClick={() => this.handleCreateNewIssue()} /> */}
```
Comment (rajat-sr) [inline client/src/components/NewIssue/NewIssue.js]: Delete this

File: client/src/components/NewIssue/NewIssue.module.css [hunk 4]
```diff
@@ -1,4 +1,8 @@
 .NewIssue {
-  width: 100%;
+  /* width: 100%; */
```
Comment (rajat-sr) [inline client/src/components/NewIssue/NewIssue.module.css]: Delete this

File: server/routes/boards-routes.js [hunk 5]
```diff
@@ -98,6 +98,7 @@ router.get('/kanban', async (req, res, next) => {
         populate: { path: 'issues', populate: { path: 'assignee' } },
       });
   } catch (e) {
+    console.log(e);
```
Comment (rajat-sr) [inline server/routes/boards-routes.js]: Delete this since we are passing it in next()

File: client/src/components/Issue/Issue.js [hunk 6]
```diff
@@ -57,19 +61,31 @@ export class Issue extends React.Component {
...
+            <Card
+              className={[classes.issue, classes.issueCard]}
+              interactive={true}
+              elevation={Elevation.TWO}
+            >
               <div onClick={() => showIssueDetails(id)}>
                 <p className={classes.issueText}>{issue}</p>
                 {allLabels}
-                <Divider />
+                {/* <Divider /> */}
```
Comment (rajat-sr) [inline client/src/components/Issue/Issue.js]: Delete this

File: client/src/components/NewIssue/NewIssue.js (hunk 3 above)
Comment (amitbadala) [reply client/src/components/NewIssue/NewIssue.js]: done!!

File: client/src/components/NewIssue/NewIssue.js (hunk 2 above)
Comment (amitbadala) [reply client/src/components/NewIssue/NewIssue.js]: done

File: client/src/components/NewIssue/NewIssue.module.css (hunk 4 above)
Comment (amitbadala) [reply client/src/components/NewIssue/NewIssue.module.css]: done

File: server/routes/boards-routes.js (hunk 5 above)
Comment (amitbadala) [reply server/routes/boards-routes.js]: done

File: client/src/components/Issue/Issue.js (hunk 6 above)
Comment (amitbadala) [reply client/src/components/Issue/Issue.js]: done

**Pull request merged**